### PR TITLE
Address 

### PR DESCRIPTION
**Pull request by seventeenthey**
all functions for address are done, including:
+ adding address
+ edit address

### CODE REVIEW

File: src/pages/OrderPage.js [hunk 1]
```diff
@@ -367,12 +371,46 @@ const OrderPage = ({ history }) => {
...
+            return (
+              <CartProduct
+                key={product.keyProductID}
+                product={product}
+                onRemove={handleRemove}
+                onQuantityChange={handleQuantityChange}
+              />
+            )
+          })
+        }
```
Comment (emanmacario) [inline src/pages/OrderPage.js:406]: Need to remove this. There is no `CartProduct` component anymore. It will make the page crash when adding a product.

Comment (emanmacario) [reply src/pages/OrderPage.js:406]: I double checked and it does crash. Need to remove.

**Comment by seventeenthey** [comment]
Hi the Address branch is done. New functions:
1. add checkout function
2. add mockup gst
3. add select,edit,add new delivery & billing addr function

**Comment by emanmacario** [comment]
Please merge master back in and fix the merge conflicts

**Review by emanmacario — APPROVED** [review]

**Pull request merged**